quality/phpstan.neon

```diff
@@ -9,6 +9,7 @@ services:
 
 parameters:
     tmpDir: var/cache/phpstan
+    reportUnmatchedIgnoredErrors: false
```

Comment:
So the error occurs only on the upper part of dependencies. As a result we have either
1. If we don't ignore the error, PHPStan is happy on the "lower" dependencies, but complains about the "regular" dependencies
2. If we ignore the error, PHP is happy on the "regular" dependencies, but complains that ignored error is unmatched at the "lower" dependencies